source/connect/cli-options.txt

```diff
@@ -0,0 +1,47 @@
+.. _cli-options:
+
+----------------------------
+Compass Command Line Options
+----------------------------
+
+The following page provides an overview on how to set command line options for 
+|compass|. For more detailed guidance on how to set a specific command line 
+option, see the task page for the specific option.
+
+Definition
+----------
+
+You can start |compass| and set configuration options from the command line. 
+When you use the command line to set configuration options, the options are 
+set at startup. Settings configured using command line options cannot be 
+modified on the |compass-short| interface.
+
+Syntax
+------
+
+Use the following syntax to start |compass| and set a configuration option:
+
+.. code-block:: sh 
+
+    <path-to-Compass-executable> --<option>
+
+Options
+-------
+
+You can set the following configuration options using the command line:
```

Comment:
Note that https://github.com/mongodb/docs-compass/pull/442 starts to document other command line options already, so whichever pull request is merged second probably needs to be updated to expand this list here cc @davemungo